akka-stream/src/main/scala/akka/stream/javadsl/FlowWithContext.scala

```diff
@@ -19,7 +18,6 @@ import scala.compat.java8.FutureConverters._
 /**
  * API MAY CHANGE
  */
-@ApiMayChange
```

Comment:
Hmm, this API is fairly new as well - even though we didn't mention it in #26532 I wonder if we should already remove the ApiMayChange here?

If we decide this API is ready to be set into stone, we should also remove the `API MAY CHANGE` scaladoc here

Reply:
@raboof If you are unsure then I'd suggest we keep it there until we are. I'm working on the scaladoc removals where relevant 👍

Reply:
ok, we can do another sweep closer to the final
(I don't think we guarantee anything for milestones, but better to avoid confusion in case we still need to change those)